File: Semester3/Advanced Programming Methods/Labs/lab3/src/domain/validators/IllegalArgumentException.java
package domain.validators;

public class IllegalArgumentException extends Exception {

    /**
     * Constructor for IllegalArgumentException class
     * @param err - error to output
     */
    public IllegalArgumentException(String err){
        super(err);
    }

    /**
     * Default Constructro for class
     */
    public IllegalArgumentException() {};
}

File: Semester4/SGBD/Tema_lab1/Lab1_CRUD_2Tables/Lab1_CRUD_2Tables/Domain/Validators/IllegalArgumentException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab1_CRUD_2Tables.Domain.Validators
{
    class IllegalArgumentException : Exception
    {

        /// <summary>
        /// Constructor for IllegalArgumentException class
        /// </summary>
        /// <param name="err">error to be thrown</param>
        public IllegalArgumentException(string err): base(err)
        {

        }

    }
}
